modules/js/generator/embindgen.py

```diff
@@ -119,6 +119,7 @@ def makeWhiteList(module_list):
     'InputOutputArray': 'cv::Mat&',
     'InputArrayOfArrays': 'const std::vector<cv::Mat>&',
     'OutputArrayOfArrays': 'std::vector<cv::Mat>&',
+    'string':'std::string', # Side effect
     'String': 'std::string',
```

Comment:
Keep existed coding style.

Please remove the comment as it doesn't help and just confuse readers.

Reply:
Thank you, i will remove the comment and fix the coding style.
If you swap the order of the entry `'string': 'std::string'` (line 122) with `'String': 'std::string'` (line 123) a double replacement happens and the code doesn't work anymore.
Should i add a comment there with a warning to not swap the order of the entries?

Reply:
Thank you for information! I will take a look on this behavior